src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1475,7 +1479,7 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader)
     {
-        if (!class_exists(PropertyAccessor::class)) {
+        if (!$container->willBeAvailable('symfony/property-access', PropertyAccessor::class, ['symfony/framework-bundle'])) {
```

Comment:
this prevents enabling `property-access` access at all for dev-only features. Shouldn't this be changed to be a configurable enabling instead (with a smart default using this logic, but still allowing an explicit choice to force it if needed) as done for other components ?

Reply:
agreed, added